### DISCUSSION TITLE
`unpaid` subscription status undocumented

### DISCUSSION THREAD

**Discussion by ba32107** [Glench/ExtPay#221] [General]
In dev mode, I can set an `unpaid` status:
![image](https://github.com/Glench/ExtPay/assets/26036493/6a4ff648-6dbd-415e-8308-1d9aacff8289)

This status is not present in the main README or this doc:
https://github.com/Glench/ExtPay/blob/main/docs/how_subscriptions_work.md

It's not really clear what the difference is between `past_due` and `unpaid`. The [Stripe docs](https://docs.stripe.com/api/subscriptions/object#subscription_object-status) say:
```
If subscription collection_method=send_invoice it becomes past_due when its invoice is not paid by the due date, and canceled or unpaid if it is still not paid by an additional deadline after that.
```

From this I can sort of understand the difference, but it's never mentioned anywhere in the ExtPay docs that the `subscriptionStatus` property mimicks the Stripe API's values. So I think this additional value should be documented in this repo.

Thanks!

**Comment by Glench** [comment]
Good call! I made an issue to update this. #223
